Makefile

```diff
@@ -121,6 +116,10 @@ update-mockgen:
 	@printf $(COLOR) "Install/update mockgen tool..."
 	@go install github.com/golang/mock/mockgen@v1.7.0-rc.1
 
+update-gotestsum:
+	@printf $(COLOR) "Install/update gotestsum..."
+	@go install gotest.tools/gotestsum@v1.11
+
```

Comment:
This is the tool [recommended by Buildkite](https://buildkite.com/docs/test-analytics/golang-collectors).